exchanges/auth/src/authExchange.ts

```diff
@@ -233,10 +234,17 @@ export function authExchange(
                 result$,
                 onStart(() => {
                   const operation = addAuthToOperation(baseOperation);
```

Comment:
it's possible that we need to remove `addAuthToOperation` here not entirely sure, it does make less sense to call this (we also say this in the docs)